### ISSUE TITLE
Incorrect line numbers

### ISSUE THREAD

**Issue by sporto** [stil4m/elm-analyse#166]
The report produced is showing the incorrect line numbers for us (one less) e.g.

```
- src/Elm/Wastage/Data.elm
  > Unnecessary parens at ((86,24),(86,64))
```

But the code looks like

![image](https://user-images.githubusercontent.com/1005498/42365242-4194c44a-8141-11e8-86fb-2adceedb21d9.png)

**Comment by stil4m** [comment]
@sporto Thanks for your report. That is currently by design, but I noticed that it has started to be confusing for a lot of end users. 
I'm planning to change it. I'll leave this ticket open for now as some personal pressure to come up with a fix.

In addition, if you are using Atom or VSCode, you may want to look into the following plugins ([1](https://atom.io/packages/elm-analyse)) ([2](https://github.com/Krzysztof-Cieslak/vscode-elm)) to get visual feedback in your editor (then you  do not  have to look at the ranges in the CLI output ;) ).

**Comment by stil4m** [comment]
This is fixed in the current master. Will be released with version  `0.16.0`
